src/ATen/Declarations.cwrap

```diff
@@ -92,6 +96,17 @@
       long_args: True
 ]]
 [[
+  name: zeros_like
```

Comment:
Can you explain what was done in this file? The change is big and it looks like a combo of merging with updated PyTorch sources (e.g. `zeros_like`) and deleting stuff to make

Reply:
1. I copied over Declarations.cwrap from PyTorch. This added a few new functions like `zeros_like` and `ones_like`
2. I combined options and used default values where possible. For example, see `kthvalue`.
3. I made `alpha` and `beta` kwarg_only in the addxxx family of functions